src/index.ts

```diff
@@ -213,8 +213,13 @@ class Client<RequestType, ResponseType> {
   }
 
   /** */
-  listStories(projectID: ID): Promise<Array<Story>> {
-    return this.listResource(`projects/${projectID}/stories`);
+  listStories(
+    projectID: ID,
+    includesDescription: boolean,
```

Comment:
Is this an optional parameter?

Reply:
Oops, nice catch!

This is working fine as `includesDescription` will be `undefined` and converted as a boolean below, but let's make it explicit to avoid confusion here.